### PR TITLE
Roslyn recommends hard selection in async completion, breaking suggetion mode

### PR DESCRIPTION
**Pull request by ivanbasov**
Fixes https://devdiv.visualstudio.com/DevDiv/_workitems/edit/787151

### CODE REVIEW

**Comment by ivanbasov** [comment]
Tagging @AmadeusW for review

File: src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb [hunk 1]
```diff
@@ -4545,6 +4545,42 @@ class C
             End Using
         End Function
 
+        <MemberData(NameOf(AllCompletionImplementations))>
+        <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TestSuggestionMode(completionImplementation As CompletionImplementation) As Task
```
Comment (dpoeschl) [inline src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb]: This really wasn't tested in Old Completion? Or is there some Old Completion-only test somewhere that wasn't updated to be used in both suites?

Comment (dpoeschl) [reply src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb]: Or was it tested in integration tests only (and Sam found it that way or something)?

Comment (ivanbasov) [reply src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb]: This really was not tested in the old completion.

**Pull request merged**